src/odemis/acq/stitching/_tiledacq.py

```diff
@@ -926,3 +928,198 @@ def acquireTiledArea(streams, stage, area, overlap=0.2, settings_obs=None, log_p
     executeAsyncTask(future, task.run)
 
     return future
+
+
+def acquireOverview(streams, stage, area, focus, ccd, overlap=0.2, settings_obs=None, log_path=None, zlevels=None,
+                    registrar=REGISTER_GLOBAL_SHIFT, weaver=WEAVER_MEAN, focusing_method=FocusingMethod.NONE):
+    """
+    Start autofocus and tiled acquisition tasks for the given list of areas.
+
+    :param streams: (list of Stream) the streams to acquire
+    :param stage: (Actuator) the stage to move
+    :param area: (list of area) area is defined by 4 floats :left, top, right, bottom positions
+     of the acquisition area (in m)
```

Comment:
for readability it's nice to have an indent for multi-line docstring argument explanations